<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<HTML>
<HEAD>
	<TITLE>Resources - CAS-AMSS Optimization & Computational Systems Biology Lab</TITLE>
</HEAD>

<BODY BGCOLOR="White" LINK="Blue" VLINK="Maroon" ALINK="Olive">
<TABLE WIDTH=device-width BORDER="0" CELLSPACING="0" CELLPADDING="10" ALIGN="CENTER" VALIGN="TOP">

<TR><TD><DIV ALIGN="center"><img src="logo.png" width="500"></DIV></TD></TR>

<TR><TD><FONT SIZE="6" COLOR="Grey"><DIV ALIGN="center">Optimization and Computational Systems Biology Lab</DIV></FONT></TD></TR>

<TR><TD>


<table width=100% cellpadding=0 cellspacing=0 border=0 >
<tr><td align=center>
<table cellpadding=10 cellspacing=0 border=0><tr><td>
<a href="index.html"><b>Research</b>
</td><td>
<a href="yongwang.html"><b>Yong Wang</b></a>
</td><td>
<a href="people.html"><b>People</b></a>
</td><td>
<a href="publications.html"><b>Publications</b></a>
</td><td>
<a href="resources.html"><b>Resources</b></a>
</td><td>
<a href="openings.html"><b>Openings</b></a>
</td>
</tr></table>
</td></tr>
</table>


<table width=100% cellpadding=0 cellspacing=0 border=0 bgcolor=aaaaaa><tr><td height=1></td></tr></table>

</TD></TR>

<TR><TD>

<b><font color="Grey"><h3>SOFTWARES</h3></font></b>


<ul>
<li><a class="wikipage" href="https://github.com/AMSSwanglab/CNEReg"><strong>CNEReg</strong>: Conserved Non-coding Element interpretation method by gene Regulatory network.</a></li>
</ul>
<TR><TD align=center><a href="https://github.com/AMSSwanglab/CNEReg"><strong><img src="CNEReg.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>


<ul>
<li><a class="wikipage" href="https://github.com/AMSSwanglab/SpecVar"><strong>SpecVar</strong>Cellular context-specific regulation-based inference of genetic correlation.</a></li>
</ul>
<TR><TD align=center><a href="https://github.com/AMSSwanglab/SpecVar"><strong><img src="SpecVar.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>


<ul>
<li><a class="wikipage" href="https://github.com/AMSSwanglab/RE-GOA"><strong>RE-GOA</strong>Annotating regulatory elements by heterogeneous network embedding.</a></li>
</ul>
<TR><TD align=center><a href="https://github.com/AMSSwanglab/RE-GOA"><strong><img src="RE-GOA.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>

<ul>
<li><a class="wikipage" href="https://github.com/AMSSwanglab/hReg-CNCC"><strong>hReg-CNCC</strong>: human Regulatory network of Cranial Neural Crest Cell (CNCC).</a></li>
</ul>
<TR><TD align=center><a href="https://github.com/AMSSwanglab/hReg-CNCC"><strong><img src="hReg-CNCC.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>


<ul>
<li><a class="wikipage" href="http://github.com/AMSSwanglab/vPECA"><strong>vPECA</strong>: Variant intepretation by Paired Expression and Chromatin Accessibility modeling</a></li>
</ul>


<TR><TD align=center><a href="http://github.com/AMSSwanglab/vPECA"><strong><img src="vPECA.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>
<ul>
<li><a class="wikipage" href="https://github.com/SUwonglab/TimeReg"><strong>TimeReg</strong>: TIME course REGulatory analysis based on paired expression and chromatin accessibility data</a></li>
</ul>

<TR><TD align=center><a href="https://github.com/SUwonglab/TimeReg"><strong><img src="TimeReg.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>

<ul>
<li><a class="wikipage" href="https://github.com/SUwonglab/PECA"><strong>PECA</strong>: Paired Expression and Chromatin Accessibility modeling</a></li>
</ul>

<TR><TD align=center><a href="https://github.com/SUwonglab/PECA"><strong><img src="PECA-s.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>

<ul>
<li><a class="wikipage" href="http://doc.aporc.org/wiki/GRNInfer"><strong>GRNInfer</strong>: A Gene Regulatory Network Inference tool from multiple microarray datasets</a></li>
</ul>

<TR><TD align=center><a href="http://doc.aporc.org/wiki/GRNInfer"><img src="GRNInfer.png" height="200"></a> </TD></TR>

</TD></TR>

<TR><TD>

<ul>

<li><a class="wikipage" href="https://github.com/AMSSwanglab/vPECA2"><strong>SpecVar</strong>:  a tool to reveal relevant tissue and associated the reulations among regulatory element and target gene for GWAS study.</a></li>


<li><a class="wikipage" href="https://github.com/AMSSwanglab/3SCover"><strong>3SCover</strong>: a method to identify safeguard TF from cell type -TF network by an extended mimimum set cover model.</a></li>


<li><a class="wikipage" href="https://github.com/AMSSwanglab/GuidingNet"><strong>GuidingNet</strong>:  a method to reveal transcriptional cofactor and predict binding for DNA methyltransferase by network regularization.</a></li>


<li><a class="wikipage" href="https://github.com/AMSSwanglab/scTIM"><strong>scTIM</strong>: A convenient tool for marker detection based on single cell RNA-seq data.</a></li>



<li><a class="wikipage" href="https://github.com/SUwonglab/DC3"><strong>DC3</strong>: Deconvolution and coupled clustering from bulk and single cell genomics data</a></li>


<li><a class="wikipage" href="https://github.com/SUwonglab/CoupledNMF"><strong>CoupledNMF</strong>: Integrative analysis of single cell genomics data by coupled nonnegative matrix factorizations</a></li>

<li><a class="wikipage" href="https://github.com/SUwonglab/PECA"><strong>PECA2</strong>: a software for inferring context specific gene regulatory network from paired gene expression and chromatin accessibility data</a></li>


<li><a class="wikipage" href="http://doc.aporc.org/wiki/EllipsoidFN"><strong>EllipsoidFN</strong>: a tool for identifying a heterogeneous set of cancer biomarkers based on gene expressions</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/GNTInfer"><strong>GNTInfer</strong>: A Gene Network reconstruction tool with compound Targets by Integrating multiple microarray datasets and prior information</a></li>


<li><a class="wikipage" href="http://doc.aporc.org/wiki/MILP_k"><strong>MILP_k</strong>: Mixed Integer Linear Programming for multiple-biomarker panel identification</a>
</li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/MNAligner"><strong>MNAligner</strong>: Alignment of molecular networks by quadratic programming</a>
</li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/Mimic"><strong>Mimic</strong>: Mixed Integer programming Model to Identify Cell type specific marker panel</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/NCCAUC"><strong>NCCAUC</strong>: Nearest Centriod Classifier for AUC valuation</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/NetPredATC"><strong>NetPredATC</strong>: Network predicting drug's anatomical therapeutic chemical code</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/PreDR"><strong>PreDR</strong>: Drug repositioning from chemical, genomic and pharmacological data in an integrated framework</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/PrePPItar"><strong>PrePPItar</strong>: Machine learning framework to Predict PPI target for drug</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/QMC"><strong>QMC</strong>: A combinatorial model and algorithm for globally searching community structure in complex networks</a></li>

<li><a class="wikipage" href="http://app.aporc.org/NOA/"><strong>NOA</strong>: Network Ontology Analysis</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/DGPsubNet"><strong>DGPsubNet</strong>: Drug-Gene-Disease coherent SUBNETworks</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/PRNA"><strong>PRNA</strong>: Prediction of protein-RNA binding sites by a random forest method with combined features</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/PSN"><strong>PSN</strong>: Predicting gene ontology functions from protein's regional surface structures</a></li>


<li><a class="wikipage" href="http://doc.aporc.org/wiki/SANA"><strong>SANA</strong>: protein Structure Alignment by Neighborhood Alignment</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/Samo"><strong>Samo</strong>: a protein structure alignment tool based on multiple objective optimization</a></li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/InferPPI"><strong>InferPPI</strong>: Predicting protein interactions based on parsimony tendency of domain interactions</a>
</li>

<li><a class="wikipage" href="http://doc.aporc.org/wiki/MDCInfer"><strong>MDCInfer</strong>: PPI prediction tool based on multiple domain cooperation analysis</a>
</li>

</ul>

</TD></TR>


<TR><TD>

<b><font color="Grey"><h3>LINKS</h3></font></b>

<ul>
<li><a href="http://zhangroup.aporc.org/">ZHANGroup in AMSS, CAS</a></li>
<li><a href="http://web.stanford.edu/group/wonglab/">Wonglab in Stanford University</a></li>
<li><a href="http://bioinfo.lab.mcgill.ca/index.html">Yu Xia's Lab in McGill University</a></li>
<li><a href="http://www.sysbio.ac.cn/cb/chenlab/">Luonan Chen's Lab in SIBS, CAS</a></li>
<li><a href="http://www.molprof.jp/~horimoto/">Katsuhisa Horimoto's group in National Institute of Advanced Industrial Science and Technology</a></li>
<li><a href="https://scholar.google.com/citations?hl=en&user=M0fnh30AAAAJ">My citation in Google Scholar</a> </li>
<li><a href="http://www.researcherid.com/rid/J-9193-2012">My Research ID</a></li>
</ul>

</TD></TR>


<TR><TD COLSPAN="2"><DIV ALIGN="center"><IMG SRC="maroon_dot.gif" WIDTH=590 HEIGHT=1 ALT="rule"
BORDER="0"><P> <FONT FACE="sans serif" SIZE="1">

Yong Wang Lab&nbsp;&nbsp;|&nbsp;&nbsp;Academy of Mathematics and Systems Science&nbsp;&nbsp;|&nbsp;&nbsp;Chinese Academy of Sciences

</FONT></DIV><P></TD></TR>

</TABLE>

</BODY>
</HTML>




